pylib/Makefile

```diff
@@ -1,5 +1,21 @@
 SHELL := /bin/bash
 FIND := $(if $(wildcard /bin/find),/bin/find,/usr/bin/find)
+MYPY_IGNORED_IMPORTS :=
```

Comment:
MYPY_ARGS might be a better, more general name

Reply:
Done.